english/snooze.md

```diff
@@ -0,0 +1 @@
+You know the alarm in your phone is put on snooze, you go back to sleep. When alarm wake you up next time after 10 minutes everything is back to normal, you don't even remember your marshmallows dream.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.